index.js

```diff
@@ -1276,6 +1276,25 @@
   }
   S.maybe = def('maybe', {}, [b, $.Function, $Maybe(a), b], maybe);
 
+  //# maybe_ :: (() -> b) -> (a -> b) -> Maybe a -> b
+  //.
+  //. Similiar to `maybe` but for use in cases where you want to compute the
+  //. Nothing case lazily. So, instead takes a value for the first argument,
+  //. it takes a function that will be called without arguments if the `Maybe`
+  //. is known to be `Nothing`.
+  //.
+  //. ```javascript
+  //. > S.maybe_(() => 0, R.length, S.Just('refuge'))
+  //. 6
+  //.
+  //. > S.maybe_(() => 0, R.length, S.Nothing)
+  //. 0
+  //. ```
```

Comment:
> For the doctest, i am using `factorial`, because when i am using fib, it doesn't fit with 79 max-line.

If you'd like to use `fib` (which is more computationally complex than `fct`, I believe), we could use the undocumented line wrapping functionality (davidchambers/doctest#67):

```javascript
> global.fib = function fib(n) {
.   return n <= 1 ? n : fib(n - 2) + fib(n - 1);
. }
fib
```